include/kickcat/Bus.h

```diff
@@ -6,6 +6,7 @@
 #include <list>
 #include <vector>
 #include <functional>
+#include <map>
```

Comment:
Prefer unordered_map unless you need to loop on the container with a sorted behavior.